.gitignore

```diff
@@ -32,6 +32,7 @@ yarn-error.log*
 .vscode
 .eslintcache
 key
+coverage
```

Comment:
What is in the coverage folder? Do I need up update my local?

Reply:
Test coverage report folder. This will address the issue.